### PR TITLE
Improve tranlations on en.json and no.json files

### PR DESCRIPTION
**Pull request by mrica-equinor**

### CODE REVIEW

File: frontend/src/language/no.json [hunk 1]
```diff
@@ -208,20 +208,20 @@
     "No predefined missions available": "Ingen tilgjengelige forhåndsdefinerte oppdrag",
     "No robot available": "Ingen robot tilgjengelig",
     "Confirm that you wish to add the selected mission(s) to the robot queue": "Bekreft at du ønsker å legge til de valgte oppdragene i robot køen",
-    "The robot is ready to run missions": "Roboten er klar til å kjøre oppdrag",
-    "The pressure measurement is not currently available, so start the mission at your own risk.": "Trykkmålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
+    "Robot ready to run missions": "Roboten er klar til å kjøre oppdrag",
```
Comment (prasm313) [inline frontend/src/language/no.json:211]:
```suggestion
    "Robot ready to run missions": "Robot klar til å kjøre oppdrag",
```

File: frontend/src/language/no.json [hunk 2]
```diff
@@ -208,20 +208,20 @@
...
+    "Pressure measurement currently unavailable, proceed with caution.": "Trykkmålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
```
Comment (andchiind) [inline frontend/src/language/no.json:212]:
```suggestion
    "Pressure measurement currently unavailable, proceed with caution.": "Trykkmåling ikke tilgjengelig, start oppdrag på egen risiko.",
```
Just to make it more similar to the English

File: frontend/src/language/no.json [hunk 3]
```diff
@@ -208,20 +208,20 @@
...
     "Confirm pressure level of robot": "Bekreft trykknivået til roboten",
-    "needs to be have a pressure level of between": "trenger å ha et trykknivå mellom",
+    "must have a pressure level between": "trenger å ha et trykknivå mellom",
```
Comment (andchiind) [inline frontend/src/language/no.json:214]:
```suggestion
    "must have a pressure level between": "må ha et trykknivå mellom",
```

File: frontend/src/language/no.json [hunk 4]
```diff
@@ -208,20 +208,20 @@
...
     "and": "og",
-    "The battery measurement is not currently available, so start the mission at your own risk.": "Batterimålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
-    "has a pressure level making it safe for the area it is operating in": "har et trykkinivå som gjør det trykt i området den kjører i",
-    "has a battery level making it safe for the area it is operating in": "har et batterinivå som gjør det trykt i området den kjører i",
-    "needs to be have a battery level greater than": "trenger å ha et batterinivå større enn",
+    "Battery measurement is currently unavailable, proceed with caution.": "Batterimålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
```
Comment (andchiind) [inline frontend/src/language/no.json:216]:
```suggestion
    "Battery measurement is currently unavailable, proceed with caution.": "Batterimåling ikke tilgjengelig, start oppdrag på egen risiko.",
```

File: frontend/src/language/no.json (hunk 2 above)
Comment (prasm313) [inline frontend/src/language/no.json:212]:
```suggestion
    "Pressure measurement currently unavailable, proceed with caution.": "Trykkmåling ikke tilgjengelig, vær klar over risikoen det medfører.",
```

File: frontend/src/language/no.json [hunk 5]
```diff
@@ -208,20 +208,20 @@
...
+    "has a safe pressure level for operation": "har et trykkinivå som gjør det trykt i området den kjører i",
+    "has a safe battery level for operation": "har et batterinivå som gjør det trykt i området den kjører i",
+    "must have a battery level greater than": "trenger å ha et batterinivå større enn",
     "Confirm battery level of robot": "Bekreft batterinivået til roboten",
     "The current pressure level is": "Trykknivået er nå",
-    "which is within the suggested range": "som er innenfor den anbefalte rekkevidden",
+    "which is within the specified range": "som er innenfor den anbefalte rekkevidden",
```
Comment (andchiind) [inline frontend/src/language/no.json:222]:
```suggestion
    "which is within the specified range": "som er innenfor den spesifiserte rekkevidden",
```

File: frontend/src/language/no.json (hunk 3 above)
Comment (prasm313) [inline frontend/src/language/no.json:214]:
```suggestion
    "must have a pressure level between": "må ha et trykknivå mellom",
```

File: frontend/src/language/no.json [hunk 6]
```diff
@@ -208,20 +208,20 @@
...
     "Warning": "Advarsel",
-    "which is NOT within the suggested range": "som IKKE er innenfor den anbefalte rekkevidden",
+    "which is NOT within the specified range": "som IKKE er innenfor den anbefalte rekkevidden",
```
Comment (andchiind) [inline frontend/src/language/no.json:224]:
```suggestion
    "which is NOT within the specified range": "som IKKE er innenfor den spesifiserte rekkevidden",
```

**Review by andchiind — COMMENTED** [review]
Just to align it with the English changes

File: frontend/src/language/no.json (hunk 4 above)
Comment (prasm313) [inline frontend/src/language/no.json:216]:
```suggestion
    "Battery measurement is currently unavailable, proceed with caution.": "Batterimåling ikke tilgjengelig, vær klar over risikoen det medfører.",
```

File: frontend/src/language/no.json [hunk 7]
```diff
@@ -208,20 +208,20 @@
...
     "Confirm pressure level of robot": "Bekreft trykknivået til roboten",
-    "needs to be have a pressure level of between": "trenger å ha et trykknivå mellom",
+    "must have a pressure level between": "trenger å ha et trykknivå mellom",
     "and": "og",
-    "The battery measurement is not currently available, so start the mission at your own risk.": "Batterimålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
-    "has a pressure level making it safe for the area it is operating in": "har et trykkinivå som gjør det trykt i området den kjører i",
-    "has a battery level making it safe for the area it is operating in": "har et batterinivå som gjør det trykt i området den kjører i",
-    "needs to be have a battery level greater than": "trenger å ha et batterinivå større enn",
+    "Battery measurement is currently unavailable, proceed with caution.": "Batterimålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
+    "has a safe pressure level for operation": "har et trykkinivå som gjør det trykt i området den kjører i",
```
Comment (prasm313) [inline frontend/src/language/no.json:217]:
```suggestion
    "has a safe pressure level for operation": "har et tilfredstillende trykkinivå",
```

File: frontend/src/language/no.json [hunk 8]
```diff
@@ -208,20 +208,20 @@
...
+    "has a safe battery level for operation": "har et batterinivå som gjør det trykt i området den kjører i",
```
Comment (prasm313) [inline frontend/src/language/no.json:218]:
```suggestion
    "has a safe battery level for operation": "har et tilfredstillende batterinivå",
```

File: frontend/src/language/no.json [hunk 9]
```diff
@@ -208,20 +208,20 @@
...
+    "must have a battery level greater than": "trenger å ha et batterinivå større enn",
```
Comment (prasm313) [inline frontend/src/language/no.json:219]:
```suggestion
    "must have a battery level greater than": "må ha et batterinivå større enn",
```

File: frontend/src/language/no.json (hunk 5 above)
Comment (prasm313) [inline frontend/src/language/no.json:222]:
```suggestion
    "which is within the specified range": "som er innenfor rekkevidden",
```

File: frontend/src/language/no.json (hunk 6 above)
Comment (prasm313) [inline frontend/src/language/no.json:224]:
```suggestion
    "which is NOT within the specified range": "som IKKE er innenfor rekkevidden",
```

File: frontend/src/language/no.json [hunk 10]
```diff
@@ -208,21 +208,21 @@
     "No predefined missions available": "Ingen tilgjengelige forhåndsdefinerte oppdrag",
     "No robot available": "Ingen robot tilgjengelig",
     "Confirm that you wish to add the selected mission(s) to the robot queue": "Bekreft at du ønsker å legge til de valgte oppdragene i robot køen",
-    "The robot is ready to run missions": "Roboten er klar til å kjøre oppdrag",
-    "The pressure measurement is not currently available, so start the mission at your own risk.": "Trykkmålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
+    "Robot ready to run missions": "Robot klar til å kjøre oppdrag",
+    "Pressure measurement currently unavailable, proceed with caution.": "Trykkmåling ikke tilgjengelig, start oppdrag på egen risiko.",
```
Comment (prasm313) [inline frontend/src/language/no.json:212]:
```suggestion
    "Pressure measurement currently unavailable, proceed with caution.": "Trykkmåling ikke tilgjengelig, vær klar over risikoen det medfører.",
```

File: frontend/src/language/no.json [hunk 11]
```diff
@@ -208,21 +208,21 @@
...
     "Confirm pressure level of robot": "Bekreft trykknivået til roboten",
-    "needs to be have a pressure level of between": "trenger å ha et trykknivå mellom",
+    "must have a pressure level between": "må ha et trykknivå mellom",
     "and": "og",
-    "The battery measurement is not currently available, so start the mission at your own risk.": "Batterimålingen er ikke tilgjengelig, så start oppdraget på egen risiko.",
-    "has a pressure level making it safe for the area it is operating in": "har et trykkinivå som gjør det trykt i området den kjører i",
-    "has a battery level making it safe for the area it is operating in": "har et batterinivå som gjør det trykt i området den kjører i",
-    "needs to be have a battery level greater than": "trenger å ha et batterinivå større enn",
+    "Battery measurement is currently unavailable, proceed with caution.": "Batterimåling ikke tilgjengelig, vær klar over risikoen det medfører.",
```
Comment (prasm313) [inline frontend/src/language/no.json:216]:
```suggestion
    "Battery measurement is currently unavailable, proceed with caution.": "Batterimåling ikke tilgjengelig, vær klar over risikoen det medfører.",
```

File: frontend/src/language/no.json [hunk 12]
```diff
@@ -208,21 +208,21 @@
...
+    "has a safe pressure level for operation": "har et tilfredstillende trykkinivå",
+    "has a safe battery level for operation": "har et tilfredstillende batterinivå",
+    "must have a battery level greater than": "må ha et batterinivå større enn",
     "Confirm battery level of robot": "Bekreft batterinivået til roboten",
     "The current pressure level is": "Trykknivået er nå",
-    "which is within the suggested range": "som er innenfor den anbefalte rekkevidden",
+    "which is within the specified range": "som er innenfor den spesifiserte rekkevidden",
```
Comment (prasm313) [inline frontend/src/language/no.json:222]:
```suggestion
    "which is within the specified range": "som er tilfredstillende",
```

File: frontend/src/language/no.json [hunk 13]
```diff
@@ -208,21 +208,21 @@
...
     "Warning": "Advarsel",
-    "which is NOT within the suggested range": "som IKKE er innenfor den anbefalte rekkevidden",
+    "which is NOT within the specified range": "som IKKE er innenfor rekkevidden",
```
Comment (prasm313) [inline frontend/src/language/no.json:224]:
```suggestion
    "which is NOT within the specified range": "som IKKE er tilfredstillende",
```

**Review by prasm313 — APPROVED** [review]
LGTM

**Pull request merged**